Spikes/movement/Pointer.cpp

```diff
@@ -0,0 +1,91 @@
+#include "stdafx.h"
+#include "Pointer.h"
+
+//using namespace sf;
+
+
+
+
+//GLOBAL VARIABLES (FOR TESTING PURPOSES)
+const int PATH_RADIUS = 200;
+const float PI = std::acos(-1);
+
+
+
+Pointer::Pointer(float radius, float speed, Color color, Vector2f centre) 
+:	m_centre(centre),
+	m_radius(radius),
+	m_angle(0),
+	m_speed(speed),
+	m_minSpeed(speed),
+	m_maxSpeed(speed * 2),
+	m_interval(speed / 60),
+	m_color(color)
+{
+	//Define shape
+	m_shape = CircleShape(m_radius);
+	m_shape.setFillColor(m_color);
+	updatePos();
+}
+
+
+void Pointer::move()
+{
+	if (Keyboard::isKeyPressed(Keyboard::LShift) && m_speed <= m_maxSpeed)
+	{
+		m_speed += m_interval;
+	}
+	else if (m_speed >= m_minSpeed)
+	{
+		m_speed -= m_interval;
+	}
+
+	if (Keyboard::isKeyPressed(Keyboard::Left))
+	{
+		m_angle -= m_speed;
+	}
+	else if (Keyboard::isKeyPressed(Keyboard::Right))
+	{
+		m_angle += m_speed;
+	}
+
+	resetAngle();
+	updatePos();
+}
```

Comment:
again does reset angles need to be called every update? If yes, just reply to clear it up